.github/workflows/ci.yaml

```diff
@@ -20,13 +20,13 @@ jobs:
     name: Initialize
     runs-on: ubuntu-20.04
     outputs:
-      target_branch: ${{ steps.target_branch.outputs.target_branch }}
-      build-branch: ${{ steps.build_branch.outputs.build_branch || steps.branch-pr.outputs.branch-pr }}
+      target_branch: ${{ env.target_branch }}
+      build_branch: ${{ env.build_branch || env.branch-pr }}
```

Comment:
I don't see `branch-pr` used anywhere - I think this is old, obsolete code. I think `|| env.branch-pr` should be removed.

Reply:
I'm removing branch-pr - this isn't set anywhere. I think this is an old remnant that is no longer used/set.